src/Metal/MTLEnums.cs

```diff
@@ -1003,5 +1003,145 @@ public enum MTLMultisampleStencilResolveFilter : ulong
 		Sample0 = 0,
 		DepthResolvedSample = 1,
 	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Flags, NoMac, NoTV, iOS (13,0)]
+	[Native, Advice ("This API is not available when using UIKit on macOS.")]
+	public enum MTLSparseTextureRegionAlignmentMode : ulong {
+		Outward = 0x0,
+		Inward = 0x1,
+	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[Flags, NoMac, NoTV, iOS (13,0)]
+	[Native, Advice ("This API is not available when using UIKit on macOS.")]
+	public enum MTLSparseTextureMappingMode : ulong {
+		Map = 0x0,
+		Unmap = 0x1,
+	}
```

Comment:
Is `0x0` just the hex representation? Is there a difference in doing `Map = 0x0` and `Map = 0` (if they're both being mapped to `ulong`?

Reply:
@prmallic yes, it's hexadecimal / identical - but it's often an indication that it's a `[Flags]` enum, i.e. that multiple values can be set (bit mask), since it makes mistake easier to spot :)